release.yml

```diff
@@ -0,0 +1,123 @@
+# JIRA Project code where the release issue should be created
+jiraProject:     SB
+
+# If already created, provide the issue number of the Release, otherwise keep it empty.
+# It will be used by the command create-component to link the stakeholder issues
+jiraKey:         # SB-1484
+
+# Version to be released. This version will be used to populate the description to be created for each Jira stakeholder: component or starter owner
+version:         2.3.0.RELEASE
+
+# Long version name used to create the title of the release issue
+longVersionName: "[Spring Boot 2.3.0] Release steps CR [Q3-2020]"
+
+# Release date scheduled. It will also be used to populate the description to be created for each Jira stakeholder: component or starter owner
+releaseDate:     September 14th 2020
+
+# Due date for the component/starter owner to release their new components, QE signoff it
+dueDate:         2020-08-21
+
+# EOL of the Snowdrop release
+eol:             September 2021
+
+components:
+  # List of JIRA Projects contributing to a Snowdrop release
+  # They will be used to create a component or starter
+  #
+  # https://issues.redhat.com/projects/KEYCLOAK/issues/
+  # https://issues.redhat.com/projects/JWS/issues
+  # https://issues.redhat.com/projects/JDG/issues
+  # https://issues.redhat.com/projects/RESTEASY/issues
+  # https://issues.redhat.com/projects/TRACING/issues
+  # https://issues.redhat.com/projects/EAPSUP/issues
+  # https://issues.redhat.com/projects/ENTMQCL/issues
+  #
+  - jiraProject: SB
+    jiraTitle: "Hibernate version to use for SB 2.3"
+    name: Hibernate
+    version: 5.4.15
+
+  - jiraProject: SB
+    jiraTitle: "Tomcat embedded version to use for SB 2.3"
+    name: Tomcat
+    version: 9.0.35
+
+  - jiraProject: SB
+    jiraTitle: "Keycloak version to use for SB 2.3"
+    name: KEYCLOAK
+    version:
+
+  - jiraProject: SB
+    jiraTitle: "Artemis version to use for SB 2.3"
+    name: Artemis
+    version: 2.12.0
+
+  - jiraProject: SB
+    jiraTitle: "Hibernate Validator version to use for SB 2.3"
+    name: Validator
+    version: 6.1.5
+
+  - jiraProject: SB
+    jiraTitle: "New RESTEasy starter for SB 2.3"
+    skipCreation: false
```

Comment:
What's that field for?

Reply:
When we will release a minor release where no jira issues should be created within the stakeholders jira project. This is a mistake for 2.3 as a jira ticket is mandatory. To be fixed

Reply:
That's not right. Stakeholders should be notified and should confirm the compatibility of their products for **all** versions we release. Otherwise, how could we be sure that they are indeed compatible? For micro-release, this should be fairly straightforward but it doesn't mean that this step should be skipped.